torch/distributed/_tensor/dispatch.py

```diff
@@ -55,6 +56,64 @@ def is_same_size_handler(
     return lhs.shape == rhs.shape
 
 
+def convolution_handler(
```

Comment:
nit: maybe we can put these two handlers in the `tp_conv.py` and import the handlers directly?

Reply:
Sure.